charts/umbrella/values.yaml

```diff
@@ -681,6 +681,200 @@ managed-identity-wallet:
       nginx.ingress.kubernetes.io/enable-cors: "true"
 
+# Set up services for a business partner pool, golden record process
+# and a gate with which the Portal and the users can share business partner data
+bpdm:
+  enabled: false
+  postgres:
+    # We use the default name for BPDM postgres
+    fullnameOverride:
+    nameOverride: bpdm-postgres
+    auth:
+      # BPDM can't handle random initial passwords at the moment
+      # so need to set a fixed one here and use it in the app configs later
+      password: &bpdmPostgresPassword "dbpasswordbpdm"
+      postgresPassword: *bpdmPostgresPassword
+    architecture: standalone
+    primary:
+      persistence:
+        enabled: false
+  keycloak:
+    # We use Central-IDP as authentication server
+    enabled: false
+  # Set up a Gate that acts as the Portal's Gate
+  bpdm-gate:
+    postgres:
+      # App uses BPDM postgres default name to find connection to the Postgres
+      fullnameOverride:
+      nameOverride: bpdm-postgres
+    ingress:
+      enabled: true
+      annotations:
+        nginx.ingress.kubernetes.io/rewrite-target: "/$2"
+        nginx.ingress.kubernetes.io/use-regex: "true"
+        nginx.ingress.kubernetes.io/x-forwarded-prefix: "/gate"
+      # The Portal expects its Gate on that specific url path so we provide it here through ingress
+      hosts:
+        - host: "business-partners.tx.test"
+          paths:
+            - path: "/gate(/|$)(.*)"
+              pathType: "ImplementationSpecific"
+    applicationConfig:
+      server:
+        # App should take the x-forward-header-prefix into account for Swagger-UI and redirects to work correctly
+        forward-headers-strategy: "FRAMEWORK"
+      bpdm:
+        bpn:
+          # This Gate has no owner restriction as other companies can write into the Gate under their own tenant
+          owner-bpn-l:
+        tasks:
+          creation:
+            fromSharingMember:
+              # Portal needs to set uploaded business partner data as ready to be shared by itself (disables setting it automatically as ready)
+              starts-as-ready: false
+        security:
+          # App's API is authenticated over Central-IDP
+          auth-server-url: "http://centralidp.tx.test/auth"
+          realm: "CX-Central"
+          # Is Central-IDPs name for the Gate client
+          # The Gate will use that name to check for existing client permissions here
+          client-id: "Cl16-CX-BPDMGate"
+        # The Gate needs to connect to the Pool and Orchestrator to realize the Golden Record Process
+        # Setup client connection for both here (where to connect and the authentication)
+        # By default the Gate assumes that the authentication server of the Pool and Orchestrator are the same as its own
+        # We will reuse the general BPDM admin technical user to establish connection between the services
+        client:
+          pool:
+            base-url: "http://business-partners.tx.test/pool"
```

Comment:
On default BPDM services try to find each other by their release name in the cluster. Since the services are created by the same Helm release and are running in the same cluster/namespace we can remove the base-url overwrite here.